serialize.h

```diff
@@ -12,14 +12,14 @@
  * a serialized system state should have SERIALIZED_OUTPUT_LEN - 1 ascii
  * characters in it
  */
-#define SERIALIZED_OUTPUT_LEN 5
+#define SERIALIZED_OUTPUT_LEN 9
 /*
  * Length of a state command. A state command is a block of characters
  * that can be sent over the radio. It's mostly the serialized state output
  * with a couple of additional bytes for STATE_COMMAND header and for a
  * CRC or hamming code
  */
-#define STATE_COMMAND_LEN 7
+#define STATE_COMMAND_LEN SERIALIZED_OUTPUT_LEN + 2
```

Comment:
Please put parentheses around the `SERIALIZED_OUTPUT_LEN + 2` so that if we ever do something like
`half_command_len = STATE_COMMAND_LEN / 2`
We get the rightvalue